Task8/src/api/video.js

```diff
@@ -0,0 +1,83 @@
+import { ActionType } from '../store/actionTypes';
+
+const API_KEY = 'AIzaSyALMiCr_df0KxwdiWxpGeQBSCrczG5RcRs';
+let nextPageToken;
+let searchString;
+
+async function loading(url,dispatch, isAppend){
+    try{
+        const request = new Request(url);
+        let res = await fetch(request);
+        let data = await res.json();
+        let video = data.items.map((video) => ({
+                imageUrlMedium: video.snippet.thumbnails.medium.url,
+                imageUrlHigh: video.snippet.thumbnails.high.url,
+                imageUrlDefault: video.snippet.thumbnails.default.url,
+                title: video.snippet.title,
+                description: video.snippet.description,
+                id: (video.id.videoId == null) ? video.id.channelId : video.id.videoId,
+            }));
+            if (isAppend) {
```

Comment:
smth wrong with indentation
should be same vertical line as  `let video`

Reply:
Refactored